kitsune/users/jinja2/users/auth.html

```diff
@@ -31,41 +31,32 @@
 
     <div class="card feature-padding elevation-03">
       <div class="sumo-auth">
-        <figure class="sumo-auth--logo">
-          <img class="ff-logo" src="{{ webpack_static('sumo/img/FF_Master__icon.svg') }}" alt="{{ _('Firefox logo') }}"/>
-        </figure>
         <div class="sumo-auth--form">
           <div id="login-fxa">
             <article id="login" class="main login-form">
-              <div class="sumo-page-section">
-                <h1 class="sumo-page-heading">{{ _('Log in / Sign up with Firefox Accounts') }}</h1>
-                <p>
-                  {% if aaq_context and not aaq_context.has_subscriptions %}
-                    {{ _('Never miss an answer. A free Firefox Account lets you view your question history and track or post responses.')}}
-                  {% else %}
-                    {% trans url=url('wiki.document', 'firefox-accounts-mozilla-support-faq') %}
-                      We've upgraded our log-in system to replace Mozilla Support accounts with Firefox Accounts.
-                      Firefox Accounts offers one log-in for convenient and secure access to Mozilla services.
-                      <a href="{{ url }}">See our FAQ</a> for more information.
-                    {% endtrans %}
-                  {% endif %}
-                </p>
-
+              <div class="sumo-page-section content-zero-bottom-pad">
+                <h1 class="sumo-page-subheading sumo-header-medium">{{ _('Continue with your Mozilla account' ) }}</h1>
+                <p class="text-body-sm">Get personalized support from staff and our<br> amazing community of contributors from around the world.</p>
```

Comment:
This should be localizable

Reply:
Inside the `{% trans` block, which apparently acts like `gettext` anyway.